README.md

```diff
@@ -36,18 +35,30 @@ Copy `.env.schema` to `.env.local` and configure it with appropriate endpoints.
 
 ### Contributing
 
-* ### Branches
+- ### Branches
 
   All new development should happen on a supporting branch rather than directly on `next` or `master`. Supporting branches should be formatted as `<type>/##-couple-words` or `<type>/very-short-description`, where <type> denotes a change type that is one of the below:
 
-  * feat (feature)
-  * bug (bug fix)
-  * docs (documentation)
-  * refactor
-  * test (when adding missing tests)
-  * chore (maintain)
+  - feat (feature)
+  - bug (bug fix)
+  - docs (documentation)
+  - refactor
+  - test (when adding missing tests)
+  - chore (maintain)
 
   Once development is complete for the scope defined by the supporting branch, a pull request can be made for the `next` branch for code review.
 
+- ### Component Development
```

Comment:
This is very helpful. Do we also want to put in a note about where storybook code lives and/or an example for starting a new component? - Probably optional for us but helpful?

Reply:
yes, can certainly add! thanks! :)